platform/android/SCsub

```diff
@@ -74,3 +76,18 @@ if lib_arch_dir != "":
         str(env["ANDROID_NDK_ROOT"]) + "/sources/cxx-stl/llvm-libc++/libs/" + lib_arch_dir + "/libc++_shared.so"
     )
     env_android.Command(out_dir + "/libc++_shared.so", stl_lib_path, Copy("$TARGET", "$SOURCE"))
+
+    def generate_apk(target, source, env):
+        subprocess.run(
+            [
+                "./gradlew",
+                "generateGodotEditor" if env["target"] == "editor" else "generateGodotTemplates",
```

Comment:
Would be good to also include support for `generateDevTemplate`.

That would be only for `target=template_debug dev_build=yes`.